make_test_refs.py

```diff
@@ -0,0 +1,34 @@
+from datasets import ClassLabel, load_dataset, Value 
+import torch
+
+
+if __name__=='__main__':
+    """
+    Creates references for evaluation and stores MRs into a file (input for a trained generative model)
+    """
+
+    data = load_dataset('e2e_nlg')['test']
+    dictic = {}
+    dataloader = torch.utils.data.DataLoader(data, batch_size=1)
+    ls=[]
+
+    for item in dataloader:
+        key = item['meaning_representation'][0]
+        value = item['human_reference'][0]
+        if key+'\n' not in ls:
```

Comment:
Why do you add the newlines everywhere (here and in the files above)? Isn't it easier and more intuitive to store it without it and then maybe add them when writing to a file?